assets/css/styles.css

```diff
@@ -379,11 +379,11 @@ img.tower {
   -ms-transform: scale(1.9);
   transform: scale(1.9);
 
-  -webkit-transform-origin : 70% 90%;
-  -moz-transform-origin : 70% 90%;
-  -o-transform-origin : 70% 90%;
-  -ms-transform-origin : 70% 90%;
-  transform-origin : 70% 90%;
+  -webkit-transform-origin : 70% 16vw;
+  -moz-transform-origin : 70% 16vw;
+  -o-transform-origin : 70% 16vw;
+  -ms-transform-origin : 70% 16vw;
+  transform-origin : 70% 16vw;
```

Comment:
What's up with the inconsistent % and vw

Stick with one, please. I suggested 70% 60% too.

Reply:
yea, broke on MBP